bubble_sort/Python/bubbleSort.py

```diff
@@ -0,0 +1,24 @@
+#Bubble Sort in Python
```

Comment:
E265 block comment should start with '# '

*Origin: PycodestyleBear (E265), Section: `all.autopep8`.*